native/cocos/audio/android/AudioEngine-inl.cpp

```diff
@@ -41,9 +44,13 @@
 #include "base/Scheduler.h"
 #include "base/UTF8.h"
 #include "application/ApplicationManager.h"
+#if CC_PLATFORM == CC_PLATFORM_ANDROID
 #include "platform/android/FileUtils-android.h"
 #include "platform/java/jni/JniHelper.h"
 #include "platform/java/jni/JniImp.h"
+#elif CC_PLATFORM == CC_PLATFORM_OPENHARMONY
+#include "cocos/platform/openharmony/FileUtils-OpenHarmony.h"
+#endif
```

Comment:
If we re-use the code in audio/android for openharmony , it's better to rename cocos/audio/android to cocos/audio/opensles

Reply:
Yes, the part including decoding can also be independent

Reply:
https://github.com/cocos/cocos-engine/issues/new